src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const currentObject = { ...state };
+  const clonesArray = [];
+
+  for (const i in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Don't iterate array using `in` operator.
Also, use descriptive names